tests/puppeteer/campaigns/functional/BO/shopParameters/contact/03_contactBulkActions.js

```diff
@@ -0,0 +1,111 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ContactsPage = require('@pages/BO/shopParameters/contact');
+const AddContactPage = require('@pages/BO/shopParameters/contact/add');
+// Importing data
+const ContactFaker = require('@data/faker/contact');
+
+let browser;
+let page;
+let numberOfContacts = 0;
+const firstContactData = new ContactFaker({description: 'todelete'});
```

Comment:
```suggestion
const firstContactData = new ContactFaker({title: 'todelete'});
```
maybe doing it with title is better